test/db/cmd/types

```diff
@@ -351,6 +351,7 @@ EXPECT_ERR=<<EOF
 Cannot find base type "xoo"
 Cannot find 'xoo' type
 Cannot find base type "xoo"
+ERROR: Cannot find "xoo" union type
```

Comment:
Why one says "ERROR:" and the others don't?

Reply:
Because other parts are in the other C files. Should I change all of them in this PR? It's out of the scope of this PR IMHO,

Reply:
Ok I thought they were coming directly from the commands. Don't like this inconsistency though :D Maybe it was better with eprintf...

\me runs away :D